Dockerfile

```diff
@@ -22,6 +24,7 @@ ENV PYTHONPATH /opt/scaife-viewer/src/
 ENV PATH="/opt/scaife-viewer/bin:${PATH}" VIRTUAL_ENV="/opt/scaife-viewer"
 WORKDIR /opt/scaife-viewer/src/
 COPY --from=static /opt/scaife-viewer/src/static/dist /opt/scaife-viewer/src/static/dist
+COPY --from=static /opt/scaife-viewer/src/webpack-stats.json /opt/scaife-viewer/src/webpack-stats.json
```

Comment:
You can get the final image even smaller by creating the wheel files in the build image and then copying them over to the final image rather than copying the entire "/opt/scaife-viewer/" directory.

Example: https://github.com/scaife-viewer/scaife-viewer/blob/onboarding-issues/Dockerfile-prod

Reply:
(We're going to defer on this until/if we retire Pipenv)